msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -157,6 +148,12 @@
     private static final String INTERNET_PERMISSION = "android.permission.INTERNET";
     private static final String ACCESS_NETWORK_STATE_PERMISSION = "android.permission.ACCESS_NETWORK_STATE";
 
+    /**
+     * Constant used to signal a home account's tenant id should be used when performing cache lookups
+     * relative to creating OperationParams.
+     */
+    private static final String FORCE_HOME_LOOKUP = "force_home";
```

Comment:
"force_home_lookup" instead?

Reply:
Resolved in b3acd1b8a29842e7c7404364ac29242f6fa273bf